core/src/main/resources/springCommon.xml

```diff
@@ -391,8 +391,10 @@
 				<prop key="DB2/NT">nl.nn.adapterframework.jdbc.dbms.Db2DbmsSupport</prop>
 				<prop key="DB2/LINUXX8664">nl.nn.adapterframework.jdbc.dbms.Db2DbmsSupport</prop>
 				<prop key="H2">nl.nn.adapterframework.jdbc.dbms.H2DbmsSupport</prop>
-				<prop key="MySQL">nl.nn.adapterframework.jdbc.dbms.MySqlDbmsSupport</prop>
+				<!-- keep MySQL out of dbmsSupport map, to let DbmsSupportFactory distinguish between MySQL and MariaDB -->
+				<!-- <prop key="MySQL">nl.nn.adapterframework.jdbc.dbms.MySqlDbmsSupport</prop> -->
 				<prop key="MariaDB">nl.nn.adapterframework.jdbc.dbms.MariaDbDbmsSupport</prop>
+				<prop key="PostgreSQL">nl.nn.adapterframework.jdbc.dbms.PostgresqlDbmsSupport</prop>
 			</props>
```

Comment:
Dit moet naar de enum toe, zie mijn comment in DbmsSupportFactory. Hartstikke leuk dat je dingen toe kan voegen via Spring, dat kan vast handig zijn, maar de 7 of 8 defaults moeten toch ergens vast liggen?